tests/mamba/conftest.py

```
@@ -1013,7 +1013,58 @@
+        if not torch.cuda.is_available():
+            return True
+        major, minor = torch.cuda.get_device_capability()
+
+        import triton
+
+        troot = os.path.dirname(triton.__file__)
+        cands = glob.glob(os.path.join(troot, "backends", "nvidia", "bin", "ptxas"))
+        if not cands:
+            return True
+        for gpu_name in (f"sm_{major}{minor}a", f"sm_{major}{minor}"):
+            r = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)